### PR TITLE
Add gateway panel in pods supervision

### PR DESCRIPTION
**Pull request by YenguiSeddik**

### CODE REVIEW

**Comment by antoinebhs** [comment]
For this to work, I had to add the gateway IP in prometheus.yaml, I think we should add it?

File: docker-compose/technical/grafana/dashboards/pods_supervision.json [hunk 1]
```diff
@@ -1126,13 +1126,213 @@
       ],
       "type": "table"
     },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "PBFA97CFB590B2093"
```
Comment (antoinebhs) [inline docker-compose/technical/grafana/dashboards/pods_supervision.json:1132]: uid should be       
```
      "uid": "${datasource}"
```

File: docker-compose/technical/grafana/dashboards/pods_supervision.json [hunk 2]
```diff
@@ -1126,13 +1126,213 @@
       ],
       "type": "table"
     },
+    {
```
Comment (antoinebhs) [inline docker-compose/technical/grafana/dashboards/pods_supervision.json:1129]: I'm not sure these gateway metrics fits in "pods supervision"? In my opinion, it should be in a separate dashboard.

**Comment by antoinebhs** [comment]
I'm not sure what's the actual difference between active vs connections, can you detail this part? 
If I understand the metrics we're monitoring the number of synchronous processing (this does not include for example asynchronous processing with queues), is it correct?

File: docker-compose/technical/grafana/dashboards/pods_supervision.json (hunk 1 above)
Comment (antoinebhs) [reply docker-compose/technical/grafana/dashboards/pods_supervision.json:1132]: I think you'll get that if you export the dashboard for external use in the share option

File: docker-compose/technical/grafana/dashboards/pods_supervision.json (hunk 2 above)
Comment (antoinebhs) [inline docker-compose/technical/grafana/dashboards/pods_supervision.json:1129]: After discussing with @SlimaneAmar , these metrics should be added to the table "Pods info and restarts".
total connections => current threads
active connections => busy threads

Comment (antoinebhs) [reply docker-compose/technical/grafana/dashboards/pods_supervision.json:1129]: there should be no timeseries for now

**Comment by antoinebhs** [comment]
> For this to work, I had to add the gateway IP in prometheus.yaml, I think we should add it?

Still not added?

File: docker-compose/technical/grafana/dashboards/pods_supervision.json [hunk 3]
```diff
@@ -834,7 +834,7 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "sum(tomcat_threads_current_threads) by (pod)",
+          "expr": "sum(tomcat_threads_current_threads) by (pod) or or sum(reactor_netty_http_server_connections) by (pod)",
```
Comment (antoinebhs) [inline docker-compose/technical/grafana/dashboards/pods_supervision.json:837]:
```suggestion
          "expr": "sum(tomcat_threads_current_threads) by (pod) or sum(reactor_netty_http_server_connections) by (pod)",
```

File: docker-compose/technical/grafana/dashboards/pods_supervision.json (hunk 2 above)
Comment (antoinebhs) [inline docker-compose/technical/grafana/dashboards/pods_supervision.json:1129]: remove these panels for now

Comment (antoinebhs) [reply docker-compose/technical/grafana/dashboards/pods_supervision.json:1129]: keep only changes above

File: docker-compose/technical/grafana/dashboards/pods_supervision.json (hunk 3 above)
Comment (YenguiSeddik) [reply docker-compose/technical/grafana/dashboards/pods_supervision.json]: done

File: docker-compose/technical/grafana/dashboards/pods_supervision.json (hunk 2 above)
Comment (YenguiSeddik) [reply docker-compose/technical/grafana/dashboards/pods_supervision.json]: done

**Review by antoinebhs — APPROVED** [review]
Code OK
Tests OK

**Pull request merged**